jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/container/TestContext.java

```diff
@@ -49,4 +53,20 @@ public String getToken() {
     public void setToken(String token) {
         this.token = token;
     }
-}
+
+    public int getCounter() {
+        return counter++;
```

Comment:
A getter shouldn't have side effects. I suggest changing name or behavior

Reply:
Added getCounterAndIncrease, could be removed in future